vellum.yaml

```diff
@@ -178,7 +178,7 @@ resources:
             #!/bin/bash
 
             # Log all output to file.
-            exec > >(tee -a /var/log/clearwater-heat-homestead.log) 2>&1
+            exec > >(tee -a /var/log/clearwater-heat-sprout.log) 2>&1
```

Comment:
We should also update the log name for dime, at https://github.com/Metaswitch/clearwater-heat/pull/40/files#diff-fbfa58039a87d980a307ef70da669b18R181